rsatoolbox/inference/result.py

```diff
@@ -67,6 +70,63 @@ def __init__(self, models, evaluations, method, cv_method, noise_ceiling,
             self.diff_var = None
             self.noise_ceil_var = None
 
+    def __repr__(self):
+        """ defines string which is printed for the object
+        """
+        return (f'rsatoolbox.inference.Result\n'
+                f'containing evaluations for {self.n_model} models\n'
+                f'evaluated using {self.cv_method} of {self.method}'
+                )
+
+    def __str__(self):
+        """ defines the output of print
+        """
+        return self.summary()
+
+    def summary(self, test_type='t-test'):
+        """
+        Human readable summary of the results
+
+        Args:
+            test_type(String):
+                What kind of tests to run.
+                See rsatoolbox.util.inference_util.all_tests for options
+        """
+        summary = f'Results for running {self.cv_method} evaluation for {self.method} '
+        summary += f'on {self.n_model} models:\n\n'
+        name_length = max([max([len(m.name) for m in self.models]) + 1, 6])
+        means = self.get_means()
+        sems = self.get_sem()
+        p_zero = self.test_zero(test_type='t-test')
```

Comment:
I believe this should be something like `self.test_zero(test_type=test_type)`, in cases where the user asks for something other than a t-test?

Reply:
yes, it should be!